### DISCUSSION TITLE
Status of Response to PDF 2.0?

### DISCUSSION THREAD

**Discussion by yukioshir** [hrynko/vue-pdf-embed#137] [Q&A]
I want to use vue-pdf-embed in vue3 to display a PDF on a browser screen.
Does vue-pdf-embed support viewing ISO 32000-2:2017 (PDF 2.0) compliant PDF documents?

**Comment by hrynko** [comment]
Hi @yukioshir,

I'm not sure. But if you have samples of such documents, could you try and share the results?

Thanks
